src/instructlab/model/backends/backends.py

```diff
@@ -219,31 +219,34 @@ def ensure_server(
         # pylint: enable=duplicate-code
     except ClientException:
         port = free_tcp_ipv4_port(host)
-        logger.debug("Using %i", port)
+        logger.debug(f"Using available port {port} for temporary model serving.")
 
         host_port = f"{host}:{port}"
         temp_api_base = get_api_base(host_port)
-        logger.debug(f"Starting a temporary server at {temp_api_base}...")
         llama_cpp_server_process = None
         vllm_server_process = None
 
         if backend == VLLM:
             # TODO: resolve how the hostname is getting passed around the class and this function
             vllm_server_process = server_process_func(port)
+            logger.info(f"Starting a temporary vLLM server at {temp_api_base}...")
             count = 0
             # TODO should this be configurable?
             vllm_startup_timeout = 300
             while count < vllm_startup_timeout:
-                sleep(1)
                 try:
                     list_models(
                         api_base=temp_api_base,
                         http_client=http_client,
                     )
-                    logger.debug(f"model at {temp_api_base} served on vLLM")
+                    logger.info(f"Successfully connected to vLLM at {temp_api_base}!")
```

Comment:
"connected to" doesn't seem appropriate vs the old debug message.  The user doesn't know why we are connecting.  Would it be better to just report the model is being served?

Reply:
Or just say "vLLM engine ready to serve at {temp_api_base}" or "vLLM engine successfully started at {temp_api_base}" maybe?

Reply:
++ okay good idea